main/guides/zoe/contract-requirements.md

```diff
@@ -103,6 +110,65 @@ Use `zcf.makeInvitation()` to create the first party's `invitation`:
     );
 ```
 
+The third argument (which is optional and wasn't needed for the first `invitation`) says
+the counterparty has to offer an `amount` matching the first party's `want.Price`,
+and shoateArgs) => {
+  ...
```

Comment:
Does not look like a valid transition from markdown prose to code. See https://1750ad84.documentation-7tp.pages.dev/guides/zoe/contract-requirements.html for how it incorrectly renders.

Reply:
Resolving all conversations that were only about the fubar contract-requirements.md